packages/amplication-server/src/core/fsStorage/fsStorage.module.ts

```diff
@@ -0,0 +1,19 @@
+import { Module } from '@nestjs/common';
+import { FsStorageService, FS_STORAGE_SERVICE } from './fsStorage.service';
+
+@Module({
+  controllers: [],
```

Comment:
Let's remove `controllers: []` as it is redundant

Reply:
Done